srcpkgs/txt2tags/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'txt2tags'
+pkgname=txt2tags
+version=3.7
+revision=1
+build_style=python3-module
+hostmakedepends="python3 python3-setuptools"
+depends="python3"
+checkdepends="tox"
+
+if [ $XBPS_LIBC = "musl" ]; then
+	checkdepends+=" python3-devel"
+fi
+
+short_desc="Tool for converting plain text into many markup languages"
+maintainer="meator <coder64@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://txt2tags.org/"
+distfiles="https://github.com/txt2tags/txt2tags/archive/refs/tags/${version}.tar.gz"
+checksum=ba745fc1aa98996578afadeb325173f7032a40f161dc4f6f7a7aa08848d11d03
+
+do_check() {
+	tox
```

Comment:
Looks line useful tests are skipped that way, and only useless are run. Please rather do following, without any checkkdepends:

    do_check() {
    	vsed -i test/lib.py -e 's:TXT2TAGS = ".*:TXT2TAGS = "build/lib/txt2tags.py":'
    	chmod u+x build/lib/txt2tags.py
    	python3 test/run.py
    }

Reply:
Ok, thanks!